plugin.video.einthusan/default.py

```diff
@@ -464,7 +464,7 @@ def preferred_server(lnk, mainurl):
 		vidpath = lnk.split('.tv/')[1]
```

Comment:
This needs to be updated to .com/. this was the reason the plugin was not working for me.

Reply:
ok. thanks. will apply fix

Reply:
fix made and commited

Reply:
let me know of any other issues during testing.

Reply:
This was working fine until two days back. It broke again today. For starters I think they now do Geo detection and take you to nearest server. I am not being redirected to einthusan.ca. second they are using cdn now for hosting binaries. I spent some time trying to get it working but it was beyond my understanding on how this script works.